admin/ucf-degree-messages.php

```diff
@@ -0,0 +1,134 @@
+            </div>
+        <?php
+            return ob_get_clean();
+        }
+
+        /**
+         * Displays a success message after update a tuition exception
+         * @author Jim Barnes
+         * @since 3.2.0
+         * @return string
+         */
+        public static function updated_tuition_success_message() {
+            ob_start();
+        ?>
+            <div class="notice notice-success is-dismissible">
+                <p>Successfully updated this program's tuition exception on the Search Service.</p>
+            </div>
+        <?php
+            return ob_get_clean();
+        }
+
+        /**
+         * Displays an error message after failing to update a tuition exception
+         * @author Jim Barnes
+         * @since 3.2.0
+         * @return string
+         */
+        public static function updated_tuition_error_message() {
+            ob_start();
+        ?>
+            <div class="notice notice-error is-dismissible">
+                <p>Failed to update this program's tuition exception on the Search Service.</p>
+                <p>Possible reasons might include:</p>
+                <ul>
+                    <li>Incorrect API key defined in the plugin options.</li>
+                    <li>Unable to contact the Search Service.</li>
+                    <li>You may not have permission to update <code>TuitionOverride</code> objects.</li>
+                </ul>
+                <p>Contact <a href="mailto:<?php echo get_option( 'admin_email' );?>"><?php echo get_option( 'admin_email' ); ?></a> for more information.</p>
+            </div>
+        <?php
+            return ob_get_clean();
+        }
+
+        /**
+         * Displays an error message after failing to contact the search service
+         * @author Jim Barnes
+         * @since 3.2.0
+         * @return string
+         */
+        public static function retrieval_error_message() {
+            ob_start();
+        ?>
+            <div class="notice notice-error is-dismissible">
+                <p>Failed to retrieve results from the search service.</p>
+                <p>Possible reasons might include:</p>
+                <ul>
+                    <li>Incorrect API key defined in the plugin options.</li>
+                    <li>Unable to contact the Search Service.</li>
+                    <li>A program with the appropriate <code>plan_code</code> and <code>subplan_code</code> mat not exist.</li>
+                </ul>
+                <p>Contact <a href="mailto:<?php echo get_option( 'admin_email' );?>"><?php echo get_option( 'admin_email' ); ?></a> for more information.</p>
+            </div>
+        <?php
+            return ob_get_clean();
+        }
+
+        /**
+         * Enqueues all messages
+         */
+        public static function enqueue_admin_notices() {
+            $screen = get_current_screen();
+
+            if ( $screen->id === 'degree' && isset( $_GET['degree_tuition_status'] ) ) {
```

Comment:
`get_current_screen()` may return `null`--make sure `$screen` is a `WP_Screen` obj before accessing its ID